index.html

```diff
@@ -5,243 +5,281 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<link rel="stylesheet" href="styles/main.css" />
+		<link rel="stylesheet" href="styles/desk-style.css" />
 		<title>Mobile portfoilo version</title>
 	</head>
 	<body>
 		<div class="container">
+			
 			<header class="portfo-header">
 				<a href="#" class="mylogo"> My Logo</a>
 				<nav class="menu-button" onclick="toggle()">
+					<ul class="desk-nav">
+						<li><a href="#">portfolio</a></li>
+						<li><a href="#">about</a></li>
+						<li><a href="#">contact</a></li>
+					</ul>
 					<img id="hamburger-btn" src="images/ham-btn.png" alt="menu" />
 				</nav>
 			</header>
 			<section class="hero">
-				<div class="hero-text">
-					<h1>I’m Beyk Glad to see you!</h1>
-					<h2> I’m a software developer! </h2>
-					<p>
-						I can help you build a product , feature or website Look through some of my work and
-						experience! If you like what you see and have a project you need coded, don’t hestiate
-						to contact me.
-					</p>
-					<a href="#">LET’S CONNECT</a>
-				</div>
-				<div class="hero-socialmedia-icons">
-					<ul>
-						<li
-							><a href="#"><img src="images/twitter.png" alt="twitter" /></a
-						></li>
-						<li
-							><a href="#"><img src="images/Linkedin_icon.svg" alt="linkdin" /></a
-						></li>
-						<li
-							><a href="#"><img src="images/medium.svg" alt="medium" /></a
-						></li>
-						<li
-							><a href="#"><img src="images/github.svg" alt="github" /></a
-						></li>
-						<li
-							><a href="#"><img src="images/angellist.svg" alt="angellist" /></a
-						></li>
-					</ul>
+				<div class="hero-content">
+					<div class="hero-text">
+						<h1>I’m Beyk 
+							<br/>Glad to see you!</h1>
+						<p>
+							I’m a software developer!, I can help you build a product , feature or website Look through some of my work and
+							experience! If you like what you see and have a project you need coded, don’t hestiate
+							to contact me.
+						</p>
+						<a href="#">LET’S CONNECT</a>
```

Comment:
- [ ] **Required:** You are required to have a clone of the template as the previous reviewer has said. I observed the follow 

Your design | Figma template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150628894-0035f993-2d80-4149-b497-2b5130dd34a6.png) | ![image](https://user-images.githubusercontent.com/27709832/150628912-76df331c-1313-4ed7-80d9-a09f903a5822.png)

All font-family should be Poppins. Let's make the change 👍 

- [ ] **Required:** Let's make the alignment similar as well on the card of projects and others, the language section should be aligned properly and the button in the contact section should be centred like all other elements in the section.

Your design | Figma template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150629086-e74f16b4-ae9b-4793-ab31-e2540640d001.png) | ![image](https://user-images.githubusercontent.com/27709832/150629113-17c2706c-cc20-469f-be84-ee65de941108.png)
![image](https://user-images.githubusercontent.com/27709832/150629167-952f5041-3390-4c22-9806-39be58526486.png) | ![image](https://user-images.githubusercontent.com/27709832/150629197-9f7920b8-c1c2-4702-a31b-6bbf5780906c.png)
![image](https://user-images.githubusercontent.com/27709832/150629278-ab05be49-3664-49b1-b127-244b168ff0cf.png) | ![image](https://user-images.githubusercontent.com/27709832/150629291-ddc69dd2-c0fd-4512-86ab-2420d371bc3b.png)